### ISSUE TITLE
About parameters & bonus 

### ISSUE THREAD

**Issue by BIGBALLON** [2017-fall-DL-training-program/Reinforcement_Learning#7]
we suggest you to use the following parameters to do training 
if you think original parameters (described in SPEC) is not good for you 

```
# ============================ Hyper Parameters ============================ #
EPISODE             = 500             # training episode
BATCH_SIZE          = 128             # batch size
GAMMA               = 0.99            # reward discount
TARGET_UPDATE_ITER  = 100             # target update frequency
MEMORY_CAPACITY     = 2000            # replay buffer size
INIT_REPLAY_SIZE    = 1000            # init replay buffer size
RECORD_VIDEO_EVERY  = 100             
MONITOR_PATH        = './vedio'       # video path
RECORD_VIDEO_FLAG   = False           # record video or not
# ============================ Hyper Parameters ============================ #

self.epsilon            = 1.0  
self.epsilon_min        = 0.001
self.epsilon_decay      = 0.999
self.learning_rate      = 0.001
```

And We suggest you change the game ``CartPole-v0`` to ``CartPole-v1`` ,
then see the performance between **DQN** and **Double DQN**

Additionally, you can modify your network's architecture and describe it in report.
for example:
```
def __init__(self, ):
    super(Net, self).__init__()
    self.fc1 = nn.Linear(N_STATES, 32)
    self.fc2 = nn.Linear(32, 32)
    self.out = nn.Linear(32, N_ACTIONS)
```

**Comment by BIGBALLON** [comment]
The results for CartPole-v0 & CartPole-v1

## CartPole-v0

- DQN

```
Episode:   45 | Reward:   18
Episode:   46 | Reward:   10
Episode:   47 | Reward:   49
Episode:   48 | Reward:   18
Episode:   49 | Reward:   13
Episode:   50 | Reward:   20
Episode:   51 | Reward:   14
Episode:   52 | Reward:   12
Episode:   53 | Reward:   39
Episode:   54 | Reward:   23
Episode:   55 | Reward:   30
Episode:   56 | Reward:   39
Episode:   57 | Reward:   48
Episode:   58 | Reward:   24
Episode:   59 | Reward:   77
Episode:   60 | Reward:   62
Episode:   61 | Reward:   98
Episode:   62 | Reward:   33
Episode:   63 | Reward:   28
Episode:   64 | Reward:   37
Episode:   65 | Reward:   60
Episode:   66 | Reward:   94
Episode:   67 | Reward:  142
Episode:   68 | Reward:  140
Episode:   69 | Reward:   99
Episode:   70 | Reward:  134
Episode:   71 | Reward:  200
Episode:   72 | Reward:  162
Episode:   73 | Reward:  200
Episode:   74 | Reward:  173
Episode:   75 | Reward:  200
Episode:   76 | Reward:  200
Episode:   77 | Reward:  200
Episode:   78 | Reward:  200
Episode:   79 | Reward:  200
Episode:   80 | Reward:  200
Episode:   81 | Reward:  192
Episode:   82 | Reward:  200
Episode:   83 | Reward:  190
Episode:   84 | Reward:  166
Episode:   85 | Reward:  170
Episode:   86 | Reward:  164
Episode:   87 | Reward:  156
Episode:   88 | Reward:  147
Episode:   89 | Reward:  165
Episode:   90 | Reward:  147
Episode:   91 | Reward:  151
Episode:   92 | Reward:  156
Episode:   93 | Reward:  163
Episode:   94 | Reward:  187
Episode:   95 | Reward:  175
Episode:   96 | Reward:  165
Episode:   97 | Reward:  188
Episode:   98 | Reward:  200
Episode:   99 | Reward:  200
Episode:  100 | Reward:  200
Episode:  101 | Reward:  200
Episode:  102 | Reward:  200
Episode:  103 | Reward:  200
Episode:  104 | Reward:  200
Episode:  105 | Reward:  200
Episode:  106 | Reward:  200
Episode:  107 | Reward:  200
Episode:  108 | Reward:  200
Episode:  109 | Reward:  200
```

- Double DQN

```
Episode:   40 | Reward:   18
Episode:   41 | Reward:   48
Episode:   42 | Reward:   15
Episode:   43 | Reward:   20
Episode:   44 | Reward:   85
Episode:   45 | Reward:   64
Episode:   46 | Reward:  107
Episode:   47 | Reward:   81
Episode:   48 | Reward:   39
Episode:   49 | Reward:  125
Episode:   50 | Reward:   18
Episode:   51 | Reward:   10
Episode:   52 | Reward:  108
Episode:   53 | Reward:  109
Episode:   54 | Reward:   74
Episode:   55 | Reward:   51
Episode:   56 | Reward:  159
Episode:   57 | Reward:   62
Episode:   58 | Reward:  160
Episode:   59 | Reward:  200
Episode:   60 | Reward:  200
Episode:   61 | Reward:  200
Episode:   62 | Reward:  200
Episode:   63 | Reward:  200
Episode:   64 | Reward:  200
Episode:   65 | Reward:  200
Episode:   66 | Reward:  200
Episode:   67 | Reward:  200
Episode:   68 | Reward:  200
Episode:   69 | Reward:  200
Episode:   70 | Reward:  200
Episode:   71 | Reward:  200
Episode:   72 | Reward:  168
Episode:   73 | Reward:  191
Episode:   74 | Reward:  200
Episode:   75 | Reward:  186
Episode:   76 | Reward:  197
Episode:   77 | Reward:  191
Episode:   78 | Reward:  180
Episode:   79 | Reward:  170
Episode:   80 | Reward:  200
Episode:   81 | Reward:  200
Episode:   82 | Reward:  200
Episode:   83 | Reward:  200
Episode:   84 | Reward:  200
Episode:   85 | Reward:  200
Episode:   86 | Reward:  200
Episode:   87 | Reward:  200
Episode:   88 | Reward:  200
Episode:   89 | Reward:  200
```

## CartPole-v1

- DQN

```
Episode:  212 | Reward:  165
Episode:  213 | Reward:  185
Episode:  214 | Reward:  177
Episode:  215 | Reward:  158
Episode:  216 | Reward:  161
Episode:  217 | Reward:  151
Episode:  218 | Reward:  175
Episode:  219 | Reward:  169
Episode:  220 | Reward:  171
Episode:  221 | Reward:  176
Episode:  222 | Reward:  176
Episode:  223 | Reward:  169
Episode:  224 | Reward:  160
Episode:  225 | Reward:  170
Episode:  226 | Reward:  184
Episode:  227 | Reward:  182
Episode:  228 | Reward:  174
Episode:  229 | Reward:  195
Episode:  230 | Reward:  231
Episode:  231 | Reward:  238
Episode:  232 | Reward:  235
Episode:  233 | Reward:  303
Episode:  234 | Reward:  500
Episode:  235 | Reward:  500
Episode:  236 | Reward:  500
Episode:  237 | Reward:  500
Episode:  238 | Reward:  435
Episode:  239 | Reward:  288
Episode:  240 | Reward:  303
Episode:  241 | Reward:  438
Episode:  242 | Reward:  500
Episode:  243 | Reward:  500
Episode:  244 | Reward:  500
Episode:  245 | Reward:  500
Episode:  246 | Reward:  500
Episode:  247 | Reward:  500
```

- Double DQN

```
Episode:   88 | Reward:  184
Episode:   89 | Reward:  190
Episode:   90 | Reward:  204
Episode:   91 | Reward:  216
Episode:   92 | Reward:  208
Episode:   93 | Reward:  299
Episode:   94 | Reward:  211
Episode:   95 | Reward:  226
Episode:   96 | Reward:  227
Episode:   97 | Reward:  229
Episode:   98 | Reward:  246
Episode:   99 | Reward:  247
Episode:  100 | Reward:  286
Episode:  101 | Reward:  259
Episode:  102 | Reward:  282
Episode:  103 | Reward:  246
Episode:  104 | Reward:  269
Episode:  105 | Reward:  215
Episode:  106 | Reward:  331
Episode:  107 | Reward:  230
Episode:  108 | Reward:  226
Episode:  109 | Reward:  227
Episode:  110 | Reward:  229
Episode:  111 | Reward:  236
Episode:  112 | Reward:  253
Episode:  113 | Reward:  277
Episode:  114 | Reward:  261
Episode:  115 | Reward:  290
Episode:  116 | Reward:  301
Episode:  117 | Reward:  330
Episode:  118 | Reward:  396
Episode:  119 | Reward:  500
Episode:  120 | Reward:  424
Episode:  121 | Reward:  395
Episode:  122 | Reward:  457
Episode:  123 | Reward:  500
Episode:  124 | Reward:  462
Episode:  125 | Reward:   85
Episode:  126 | Reward:  105
Episode:  127 | Reward:  240
Episode:  128 | Reward:  500
Episode:  129 | Reward:  500
Episode:  130 | Reward:  500
Episode:  131 | Reward:  500
Episode:  132 | Reward:  500
Episode:  133 | Reward:  500
Episode:  134 | Reward:  500
Episode:  135 | Reward:  500
Episode:  136 | Reward:  500
Episode:  137 | Reward:  492
Episode:  138 | Reward:  500
Episode:  139 | Reward:  500
Episode:  140 | Reward:  500
Episode:  141 | Reward:  500
Episode:  142 | Reward:  500
```

**Comment by yyrkoon27** [comment]
Dear TA,

What kind of network architecture and hyper-parameters do you use when generating the v0 and v1 results? They climb up to full reward so fast!

**Comment by BIGBALLON** [comment]
Hi, @yyrkoon27 
all of the hyper-parameters and  network architecture I used are described in this Issue. 
See the details in the first comment.
